content/docs/esc/environments/imports.md

```diff
@@ -11,12 +11,16 @@ menu:
     weight: 2
 ---
 
-Environments can also be composed from other environments.
+Environments can be composed from other environments.
 
 Different applications are often configured in similar ways and with common values --- for example, an e-commerce site and order-management system both configured to use the same cloud account, database-connection string, and third-party API key. Managing the duplication of these values across multiple configuration files, however, can be difficult, especially when one of those values changes --- e.g., when an API key is regenerated.
 
 To address these challenges, Pulumi ESC allows you to identify common or closely related configuration settings and define them only once, as individual environments, and then _import_ those environments into other, more specialized environments as needed. Imports also allow you to expose certain environments without having to distribute any concrete values and to delegate responsibility for particular environments to other teams in your organization. Environments can import both static and dynamic values, including secrets, from any number of other environments.
 
+## Explicit imports
+
+Explicit imports are defined in the `imports` section of an environment. The `imports` section is a list of environment paths that are resolved at runtime. The values from the imported environments are merged into the current environment, with the current values overwriting the imported environment's values where keys are redefined.
```

Comment:
Do we document the merge process more completely? Probably worth mentioning that it's [JSON Merge Patch](https://www.rfc-editor.org/rfc/rfc7396).